runtime/centrifuge/src/migrations.rs

```diff
@@ -195,404 +208,316 @@ mod asset_registry {
+						symbol: b"xcUSDC".to_vec(),
+						existential_deposit: 10_000u128,
+						location: None,
+						additional: CustomMetadata {
+							mintable: false,
+							permissioned: false,
+							pool_currency: true,
+							transferability: CrossChainTransferability::Xcm(XcmMetadata {
+								fee_per_second: None,
+							}),
+						},
+					},
+				),
+				(
+					CurrencyId::ForeignAsset(3),
+					orml_asset_registry::AssetMetadata {
+						decimals: 12,
+						name: b"Acala Dollar".to_vec(),
+						symbol: b"aUSD".to_vec(),
+						existential_deposit: 10_000_000_000u128,
+						location: Some(VersionedMultiLocation::V3(MultiLocation::new(
+							1,
+							Junctions::X2(
+								Parachain(2000),
+								GeneralKey {
+									length: 2,
+									data: gk,
+								},
+							),
+						))),
+						additional: CustomMetadata {
+							mintable: false,
+							permissioned: false,
+							pool_currency: true,
+							transferability: CrossChainTransferability::Xcm(XcmMetadata {
+								fee_per_second: None,
+							}),
+						},
+					},
+				),
+				(
+					CurrencyId::ForeignAsset(4),
+					orml_asset_registry::AssetMetadata {
+						decimals: 18,
+						name: b"Glimmer".to_vec(),
+						symbol: b"GLMR".to_vec(),
+						existential_deposit: 1_000_000_000_000_000u128,
+						location: Some(VersionedMultiLocation::V3(MultiLocation::new(
+							1,
+							Junctions::X2(Parachain(2004), PalletInstance(10)),
+						))),
+						additional: CustomMetadata {
+							mintable: false,
+							permissioned: false,
+							pool_currency: false,
+							transferability: CrossChainTransferability::Xcm(XcmMetadata {
+								fee_per_second: None,
+							}),
+						},
+					},
+				),
+				(
+					CurrencyId::ForeignAsset(5),
+					orml_asset_registry::AssetMetadata {
+						decimals: 10,
+						name: b"DOT".to_vec(),
+						symbol: b"DOT".to_vec(),
+						existential_deposit: 100_000u128,
+						location: Some(VersionedMultiLocation::V3(MultiLocation::new(
+							1,
+							Junctions::Here,
+						))),
+						additional: CustomMetadata {
+							mintable: false,
+							permissioned: false,
+							pool_currency: false,
+							transferability: CrossChainTransferability::Xcm(XcmMetadata {
+								fee_per_second: None,
+							}),
+						},
+					},
+				),
+				(
+					CurrencyId::ForeignAsset(6),
+					orml_asset_registry::AssetMetadata {
+						decimals: 6,
+						name: b"USD Coin".to_vec(),
+						symbol: b"USDC".to_vec(),
+						existential_deposit: 1000_u128,
+						location: Some(VersionedMultiLocation::V3(MultiLocation::new(
+							1,
+							X3(Parachain(1000), PalletInstance(50), GeneralIndex(1337)),
+						))),
+						additional: CustomMetadata {
+							mintable: false,
+							permissioned: false,
+							pool_currency: true,
+							// TODO(@review): Should probably be LiquidityPoolsTransferability?
+							transferability: CrossChainTransferability::Xcm(XcmMetadata {
+								fee_per_second: None,
+							}),
```

Comment:
No, that is perfect as is. We only want to receive that via xcm into accounts for now. I think we first need to go over the implications of having both transferabilities enabled for a single asset.